tsconfig.json

```diff
@@ -3,7 +3,12 @@
     "target": "ES2018",
     "module": "ESNext",
     "moduleResolution": "Node",
-    "lib": ["ESNext", "ESNext.AsyncIterable", "DOM"],
+    "lib": [
+      "ESNext",
+      "ESNext.AsyncIterable",
+      "DOM",
+      "webworker"
```

Comment:
This should not be part of the main config, but live in its own sub-config specific to workers, since it collides with around 16 DOM types.